lib/PostMessageTransport.js

```diff
@@ -83,6 +95,13 @@ define(function (require, exports, module) {
             PostMessageTransportRemote +
             "</script>\n";
     }
+
+    function reload() {
+        var launcher = Launcher.getCurrentInstance();
+        var liveDoc  = LiveDevMultiBrowser._getCurrentLiveDoc();
+
+        launcher.launch(liveDoc.doc.file.fullPath);
```

Comment:
Add a comment here to explain how `fullPath` is exposed for testing. This could change in the future and break this logic